package/src/components/ImageGallery/components/AnimatedGalleryVideo.tsx

```diff
@@ -156,6 +156,21 @@ export const AnimatedGalleryVideo = React.memo(
       };
     }, [previous, selected]);
 
+    const animatedStyles = useAnimatedStyle(() => {
```

Comment:
Total nitpick and not something that needs to be done now, but - the same (complex) logic is in 2 places. Perhaps a hook that calculates this would be nice

Reply:
Done ✅